indico/htdocs/js/indico/jquery/categorypicker.js

```diff
@@ -0,0 +1,332 @@
+            self.$categoryNavigator = $('<div>', {class: 'category-navigator i-box just-group-list with-hover-effect'})
+                .append($('<div>', {class: 'i-box-content'})
+                    .append(self.$category)
+                    .append(self.$categoryTree)
+                    .append(self.$categoryResults));
+            self.element.append(self.$categoryNavigator);
+
+            if (self.options.height) {
+                self.$category.closest('.category-picker').height(self.options.height);
+            }
+        },
+
+        _createSearchField: function() {
+            var self = this;
+
+            // Visible search field
+            self.$searchInput = $('<input>', {
+                class: 'js-search-category js-typeahead',
+                type: 'search',
+                name: 'q',
+                placeholder: $T.gettext("Search")
+            }).attr('autocomplete', 'off');
+
+            // Insert elements in DOM
+            var $typeaheadForm = $($.parseHTML(self._typeaheadTemplate)[0]);
+            $typeaheadForm.find('.typeahead__query').append(self.$searchInput);
+            self.element.prepend($typeaheadForm);
+
+            // Typeahead init
+            self.$searchInput.typeahead({
+                dynamic: true,
+                filter: false,
+                source: {
+                    ajax: {
+                        url: build_url(Indico.Urls.Categories.search),
+                        path: 'categories',
+                        data: {
+                            q: "{{query}}"
+                        }
+                    }
+                },
+                resultContainer: false,
+                cancelButton: false,
+                display: 'title',
+                callback: {
+                    onClickAfter: function(node, a, item) {
+                        self.goToCategory(item.id);
+                    },
+                    onResult: function(node, query, result) {
+                        self._renderResults(result);
+                    },
+                    onHideLayout: function() {
+                        self.$categoryResults.html('');
+                        self.$category.show();
+                        self.$categoryTree.show();
+                    },
+                    onShowLayout: function(node, query) {
+                        self.$category.hide();
+                        self.$categoryTree.hide();
+                    }
+                }
+            });
+        },
+
+        _buildBreadcrumbs: function(path, clickable) {
+            var self = this;
+            var $breadcrumbs = $('<ul>', {class: 'breadcrumbs'});
+            var tag = clickable? '<a>' : '<span>';
+
+            _.each(path, function(category) {
+                var $item = $(tag, {
+                    text: category.title
+                });
+                if (clickable) {
+                    $item.attr('href', '');
```

Comment:
no strong opinion, just wanted to point it out since initially you were setting two attributes